app/javascript/components/upload/AnnDataStep.jsx

```diff
@@ -20,16 +21,30 @@ export default {
 
 /** Renders a form for uploading one or more AnnData files */
 function AnnDataForm({
+  serverState,
   formState,
   addNewFile,
   updateFile,
   saveFile,
   deleteFile
 }) {
   const AnnDataFiles = formState.files.filter(AnnDataFileFilter)
+  const featureFlagState = serverState.feature_flags
+  console.log('featureflagstate:', featureFlagState)
+  console.log('featureflagstate ingest_anndata:', featureFlagState?.ingest_anndata_file)
+
+  // if the feature flag is flipped to ingest the AnnData file update the file_type
+  let NEW_ANNDATA_FILE = DEFAULT_NEW_ANNDATA_FILE
+  if (featureFlagState?.ingest_anndata_file) {
+    NEW_ANNDATA_FILE = {
+      file_type: 'AnnData Ingestible',
```

Comment:
Blocker: per other comment:

```suggestion
      file_type: 'AnnData',
```